homeassistant/components/overkiz/climate_entities/atlantic_electrical_heater_with_adjustable_temperature_setpoint.py

```diff
@@ -48,6 +50,7 @@
     OverkizCommandParam.AUTO: HVACMode.AUTO,
     OverkizCommandParam.BASIC: HVACMode.HEAT,
     OverkizCommandParam.STANDBY: HVACMode.OFF,
+    OverkizCommandParam.EXTERNAL: HVACMode.AUTO,
     OverkizCommandParam.INTERNAL: HVACMode.AUTO,
```

Comment:
It's nothing but you can keep the same order between internet & external on `OVERKIZ_TO_PRESET_MODE` & `OVERKIZ_TO_HVAC_MODE`, the order you want.